problems/4.arrays.js

```diff
@@ -2,7 +2,11 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
+function firstAndLastAverage(arr) {
+ let totalaverage = [arr[arry.length -1]]
+ return arry.length[0] / 2;
+
```

Comment:
Mixing up some things. 
arr.length gives us the length which is number. 
First element is arr[0] , last is arr[arr.length - 1] Average would be those numbers added together and divided by 2